hiclass/HierarchicalClassifier.py

```diff
@@ -96,6 +105,8 @@ def __init__(
             If True, skip scikit-learn's checks and sample_weight passing for BERT.
         classifier_abbreviation : str, default=""
             The abbreviation of the local hierarchical classifier to be displayed during logging.
+        calibration_method : {"ivap", "cvap", "platt", "isotonic"}, str, default=None
```

Comment:
Maybe there is a better way to represent the multiple possible values here. Perhaps Union["ivap", "cvap", "platt", "isotonic"]? I am not so sure without building the documentation to see the result.

Reply:
I tried building the documentation. This approach should work.